spec/requests/readings_spec.rb

```diff
@@ -0,0 +1,63 @@
+require "spec_helper.rb"
+
+describe "readings API" do
+  let(:sensor_name) {  "0013a20040c17f44" }
+  let(:reading_params) do
+    {
+      "reading" => {
+        "sensor_name" => sensor_name,
+        "temp" => "108.06",
+        "time" => "1453991049.0",
+        "verification" => "c0ffee"
+      }
+    }
+  end
+
+  it "returns 404 for readings with no sensor" do
+    post "/readings.json", reading_params
+
+    expect(json).to eq ({
+      "code" => 404,
+      "error" => "No sensor by that name found"
+    })
+    expect(response.code).to eq 404
+  end
+
+  it "returns 404 and error message for readings with no user" do
+    create(:sensor, name: sensor_name)
+
+    post "/readings.json", reading_params
+    expect(json).to eq({
+      "code" => 404,
+      "error" => "No user associated with that sensor"
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.